aviatrix/provider.go

```diff
@@ -146,6 +146,7 @@ func Provider() *schema.Provider {
 			"aviatrix_firenet_vendor_integration":       dataSourceAviatrixFireNetVendorIntegration(),
 			"aviatrix_gateway":                          dataSourceAviatrixGateway(),
 			"aviatrix_gateway_image":                    dataSourceAviatrixGatewayImage(),
+			"aviatrix_all_network_domains":              dataSourceAviatrixAllNetworkDomains(),
```

Comment:
Insert new resources/data sources alphabetically. This should be moved to below `aviatrix_account`.